### DISCUSSION TITLE
IntelliJ codestyle?

### DISCUSSION THREAD

**Discussion by bfrggit** [Netflix/conductor#2940] [General]
We are maintaining an internal fork of this project and have some customizations on top of an old version. Now someone reformatted between old and new and when we upgrade we are facing huge amount of merge conflicts. I am wondering what is the codestyle config that the project is using? Can you add the IntelliJ codestyle config XML to the repo so all contributors agree on the same style?

**Comment by jxu-nflx** [comment]
Hello @bfrggit conductor uses the spotless plugin for code format: https://github.com/Netflix/conductor/blob/main/build.gradle#L180
You could run `./gradlew spotlessApply` to fix the format. As for Intellij, you can try the Intellij plugin listed in their [github page](https://github.com/diffplug/spotless): https://plugins.jetbrains.com/plugin/18321-spotless-gradle
